src/react/components/icons/LayerIcon.jsx

```diff
@@ -24,10 +24,10 @@ export const LayerIcon = ({ type, hasTimeseries = false, additionalTooltipKey, .
         return <DataLayerIcon {...rest} />;
     };
 
-    let tooltipTitle = (<Message messageKey={ `layerTooltipTitle.${type}` } />);
+    let tooltipTitle = (<Message messageKey={ `layerTooltipTitle.${type}`  } bundleKey='LayerList' />);
```

Comment:
We should move the tooltips to the generic translations instead of linking them from under LayerList if the icons are here.